presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java

```diff
@@ -160,12 +154,7 @@ String selectSql(TableId table, List<String> requiredColumns)
     String selectSql(TableId table, String formattedColumns, String[] filters)
```

Comment:
`String[] filters` become unused var after this change. 

JFY: I'm looking into if we can revive `createWhereClause` method, but you don't need to care my work.